test/test-fixtures/transformations/resolved html/p-applicant-british-citizen-or-eu-national.js

```diff
@@ -370,11 +370,7 @@ const html = `<!DOCTYPE html>
     
   
 
-<button type="button" class="govuk-button govuk-modal__continue" data-module="govuk-button">
-  Continue application
-</button>
-
-    <!-- <a href="" class="govuk-modal-dialogue__cancel govuk-body govuk-link govuk-link--no-visited-state">Start again</a> -->
+                    <a href="#" role="button" draggable="false" class="govuk-button govuk-modal__continue ga-event--click" tracking-category="modal-button" data-module="govuk-button">Continue application</a>
```

Comment:
Check if data-* should be used... GDS design system might provide a mechanism for data-* and allow buttons to be used if it's easier